config/routes.rb

```diff
@@ -94,6 +94,13 @@
         end
       end
 
+      resources :exam_templates do
+        member do
+          get 'download'
+          post 'update'
```

Comment:
This is not necessary. `resources` will automatically create an `upload` route (see the Rails "Routing" guide), which you can use when you set up the form properly